Packs/SAPCloudForCustomerC4C/pack_metadata.json

```diff
@@ -13,6 +13,14 @@
     "useCases": [],
     "keywords": [],
     "marketplaces": [
-        "marketplacev2"
+        "marketplacev2",
+        "platform"
+    ],
+    "supportedModules": [
+        "X1",
+        "X3",
+        "X5",
```

Comment:
@Shir2611  - Are you sure we should add all those modules? 
Please verify that.